### PR TITLE
feat: migrating space page buttons and wrappers to Mantine

### PR DESCRIPTION
**Pull request by yasmin-e**
Closes: 

### Description:
migrated to Mantine
```
wrappers
breadcrumbs
Share button
Three dots button
```
refactored
```
Page header components (i.e. breadcrumbs, share button, three dots button) 
previously in SpacePanel are now in SpacePage
```

Screenshot 2023-03-21 at 13 01 54
Screenshot 2023-03-21 at 13 02 19

### CODE REVIEW

File: packages/frontend/src/pages/Space.tsx [hunk 1]
```diff
@@ -1,57 +1,163 @@
...
+                                projectUuid,
+                            })}
+                        >
+                            <ShareSpaceModal
+                                space={space!}
+                                projectUuid={projectUuid}
+                            />
+                            {renderSpaceBrowserMenu()}
+                            {renderUpdateSpaceModal()}
+                            {renderDeleteSpaceModal()}
```
Comment (IrakliJani) [inline packages/frontend/src/pages/Space.tsx:154]: not a big fan of render methods. any particular reason why we might need them? it's just a lot of back and forth when reading the code

File: packages/frontend/src/pages/Space.tsx [hunk 2]
```diff
@@ -1,57 +1,163 @@
...
+        );
+    };
+
     return (
-        <Page>
+        <Center my="md">
             <Helmet>
-                <title>{data?.name} - Lightdash</title>
+                <title>{space?.name} - Lightdash</title>
             </Helmet>
```
Comment (IrakliJani) [inline packages/frontend/src/pages/Space.tsx:126]: this should probably go before `<Center my="md">`...

File: packages/frontend/src/components/SpacePanel/index.tsx [hunk 3]
```diff
@@ -298,7 +201,7 @@ export const SpacePanel: React.FC<Props> = ({ space }) => {
                     setIsCreateDashboardOpen(false);
                 }}
             />
-        </Stack>
+        </>
```
Comment (IrakliJani) [inline packages/frontend/src/components/SpacePanel/index.tsx:204]: extra fragment wrapper

Comment (IrakliJani) [reply packages/frontend/src/components/SpacePanel/index.tsx:204]: im happy to get rid of this file if space panel is not referenced more than once

**Review by IrakliJani — CHANGES_REQUESTED** [review]
looks good, requested some minor changes and left few questions.

File: packages/frontend/src/pages/Space.tsx (hunk 1 above)
Comment (yasmin-e) [reply packages/frontend/src/pages/Space.tsx:154]: just thought it'd make things tidier but i can get rid of them

File: packages/frontend/src/components/SpacePanel/index.tsx (hunk 3 above)
Comment (yasmin-e) [reply packages/frontend/src/components/SpacePanel/index.tsx:204]: sounds good! its only refrenced here but i was afraid itd make the space page too big 
if its ok with you though ill scrap it

Comment (yasmin-e) [reply packages/frontend/src/components/SpacePanel/index.tsx:204]: ok i'll scrap it in 2nd PR cause I'm making changes to the ResourceView component there

File: packages/frontend/src/pages/Space.tsx [hunk 4]
```diff
@@ -63,101 +63,94 @@ const Space: FC = () => {
...
-                    </PageBreadcrumbs>
-                    <Group spacing="xs">
-                        <Can
-                            I="manage"
-                            this={subject('Space', {
-                                organizationUuid: user.data?.organizationUuid,
-                                projectUuid,
-                            })}
+            <Center my="md">
+                <Stack spacing="xl" w={900}>
```
Comment (IrakliJani) [inline packages/frontend/src/pages/Space.tsx:72]: can we specify width="100%" here or something? I believe page has a some kind of wrapper which is 900px? I might be wrong just asking if you know more about it

**Review by IrakliJani — APPROVED** [review]

**Pull request merged**